modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -202,6 +204,66 @@ public Error OpenInExternalEditor(Script script, int line, int col)
                 case ExternalEditorId.None:
                     // Not an error. Tells the caller to fallback to the global external editor settings or the built-in editor.
                     return Error.Unavailable;
+                case ExternalEditorId.CustomEditor:
+                {
+                    string file = ProjectSettings.GlobalizePath(script.ResourcePath);
+                    var execCommand = _editorSettings.GetSetting(Settings.CustomExecPath).As<string>();
+                    var execArgs = _editorSettings.GetSetting(Settings.CustomExecPathArgs).As<string>();
+                    var args = new List<string>();
+                    var from = 0;
+                    var numChars = 0;
+                    var insideQuotes = false;
+                    var hasFileFlag = false;
+
+                    for (int i = 0; i < execArgs.Length; ++i)
```

Comment:
Not sure what exactly the problem is, but when using the args `"{file}:{line}"` the global one calls `C:\dir\test\msgbox.exe "C:/dir/Net7/new folder/new_script.gd:0"` while this breaks: `"C:/dir/test/msgbox.exe" "C:/dir/Net7/new folder/new_script.cs"`. (some other things are funky as well when quoting arguments)